tests/cases/fourslash/completionListAtIdentifierDefinitionLocations_interfaceMembers.ts

```diff
@@ -1,7 +0,0 @@
-/// <reference path='fourslash.ts' />
-
-////var aa = 1;
-
-////interface a { /*interfaceValue1*/
-
-goTo.eachMarker(() => verify.completionListIsEmpty());

```

Comment:
can you instead of deleting this have test case that verifies "aa" is not present in the completion

Reply:
`completionsInterfaceElement` does basically the same thing -- `verify.completionsAt` will fail if there were completions we didn't list.